### ISSUE TITLE
Custom archive page layout in Elementor Pro is not working

### ISSUE THREAD

**Issue by codeagencybe** [oceanwp/oceanwp#127]
Hello Nicolas,

I have created a custom single product layout and custom archive layout. 
The single product page layout from Elementor is working perfectly with OceanWP, but the archive page not. 
Any idea why? And how to fix this? 
I know Elementor Pro has a warning that themes can override the layout from Elementor, but I find it strange that it works fine for single page but not archive layouts.... 

Please see the issue here with screen recording:
http://sharing.codeagency.be/e704423a8cd0

Everything is updated to most newest version
It's a sandbox server and site to play around with new updates and features, so no urgency here. 

Active plugins (44)
--
Advanced Custom Fields PRO |   | by Elliot Condon – 5.7.3
ATUM Inventory Management for WooCommerce |   | by Stock Management Labs™ – 1.4.13
Custom Post Type Page Template |   | by Hiroaki Miyashita – 1.1
Custom Post Type Archives |   | by Ben Irvin – 1.1
Custom Post Type UI |   | by WebDevStudios – 1.5.8
Dashboard Welcome for Elementor |   | by IdeaBox Creations – 1.0.4
Duplicate Post |   | by Enrico Battocchi – 3.2.2
Dynamic Visibility for Elementor |   | by Dynamic.ooo – 1.1.0
Ele Custom Skin |   | by Liviu Duda – 1.0.5
Elementor Extras |   | by Namogo – 1.9.14
Elementor Pro |   | by Elementor.com – 2.1.6
Elementor |   | by Elementor.com – 2.2.0
Flexible Elementor Panel |   | by Creative Web Solution – 1.7.0
iThemes Security Pro |   | by iThemes – 5.4.8
JetBlocks For Elementor |   | by Zemez – 1.1.0
JetBlog For Elementor |   | by Zemez – 2.1.6
Jet Elements For Elementor |   | by Zemez – 1.12.2
JetEngine |   | by Zemez – 0.9.0-beta1
JetMenu |   | by Zemez – 1.4.4
Jet Plugins Wizard |   | by Zemez – 1.1.0
JetReviews For Elementor |   | by Zemez – 1.1.1
Jet Tabs For Elementor |   | by Zemez – 1.1.1
JetThemeCore |   | by Zemez – 1.1.1
Jet Tricks |   | by Zemez – 1.1.1
JetWooBuilder For Elementor |   | by Zemez – 1.2.0 – Not tested with the active version of WooCommerce
Central Color Palette |   | by Gáravo – 1.12.5
Lenix Elementor Leads addon |   | by Lenix – 1.3
Media Cleaner |   | by Jordy Meow – 5.0.2
Ocean Cookie Notice |   | by OceanWP – 1.0.4
Ocean Elementor Widgets |   | by OceanWP – 1.1.0
Ocean Extra |   | by OceanWP – 1.4.23
Ocean Hooks |   | by OceanWP – 1.1.1
Ocean Popup Login |   | by OceanWP – 1.0.5
Ocean Portfolio |   | by OceanWP – 1.1.4
Ocean Side Panel |   | by OceanWP – 1.0.11
Ocean Sticky Footer |   | by OceanWP – 1.0.9
Ocean Sticky Header |   | by OceanWP – 1.1.11
Ocean White Label |   | by OceanWP – 1.0.5
Ocean Woo Popup |   | by OceanWP – 1.0.6
PowerPack Elements |   | by Team IdeaBox - PowerPack Elements – 1.3.5.1
Widget Options |   | by Phpbits Creative Studio – 3.7.1
WooCommerce Live Checkout Field Capture |   | by Streamline.lv – 2.0.3
WooCommerce |   | by Automattic – 3.4.4

**Comment by nicolas-owp** [comment]
Hello, unfortunately, I cannot solve this issue because Elementor uses their own selectors, this is why I have created a Woo Products widget in Ocean Elementor Widgets, so it is fully compatible with OceanWP.

**Comment by codeagencybe** [comment]
Hello Nicolas,

Sorry, I don't understand what that Woo Products widget does in this case? 
I have created a custom products archive layout in Elementor Pro. 
I have added your Woo Products widget like you mention, but still I keep seeing the default OWP archive layout. 
So how does that new widget from OWP solve this or give me a workaround? Because no matter what I try, it's impossible to load the custom archive layout. 

But how come it does work for the single product but not for the archive pages? 
I have also JetWooBuilder plugin from Zemez and that one does work for archive pages, so it must be possible if they can do it. 
You can check it here: https://jetwoobuilder.zemez.io/


Basically what I'm trying to do is adding some custom fields to the archive overview like "brand" and a custom badge for delivery delay and add some nice hover effects (drop shadow on hover) and make the box grow bigger on hover and show extra fields like this effect:

![screen recording 2018-09-02 at 07 55 pm](https://user-images.githubusercontent.com/7568546/44959123-2f10a380-aeea-11e8-8a2b-44168625a99d.gif)

**Comment by nicolas-owp** [comment]
Ah okay, I taught the issue was the product styling, can you create a pre-purchase ticket on oceanwp.org? I will ask you personal information to see what can be wrong.
